### ISSUE TITLE
Short description is cut on FSE themes

### ISSUE THREAD

**Issue by carolframen** [woothemes/woocommerce#44185]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

If the short description of a product is not actually short, it is cut on FSE themes, while on classic themes it shows in full regardless of the length. 

### Expected behavior

For the description to either:
- Show in full like on classic themes
- Have a read more button, so customers could see it all
- Or for the user to not be able to add a long description there, to avoid cutting it.

Here are screenshots from a classic theme (Storefront, I also tested with Twenty Nineteen):

![storefront 2](https://github.com/woocommerce/woocommerce/assets/119800062/b2d0d7f0-cb7b-4939-9320-d4d67966e544)
![storefront](https://github.com/woocommerce/woocommerce/assets/119800062/af4c200a-d29a-4e10-829a-567b0489e4b5)

### Actual behavior

The description is cut in the middle of the text and it just adds ellipses to it.

Here are screenshots from a FSE theme (Twenty Twenty-Two, but I also tested with Twenty Twenty Three and Twenty Twenty Four)

![2022](https://github.com/woocommerce/woocommerce/assets/119800062/d8c428c4-fcc8-4080-974b-0ff526438f6a)
![2022 2](https://github.com/woocommerce/woocommerce/assets/119800062/b959e0e8-f7c3-4ff0-a68c-960bd7c4c55f)

### Steps to reproduce

1. Activate a FSE theme
2. Add a long description on the short description section and update the product
3. Check the product page.

### WordPress Environment

<details>

```
### WordPress Environment ###

WordPress address (URL): https://carol-testing.mystagingwebsite.com
Site address (URL): https://carol-testing.mystagingwebsite.com
WC Version: 8.5.2
REST API Version: ✔ 8.5.2
WC Blocks Version: ✔ 11.8.0-dev
Action Scheduler Version: ✔ 3.7.1
Log Directory Writable: ✔
WP Version: 6.4.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 8.2.15
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 8.4.0
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 10.6.15-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.5.2
WC Database Prefix: wp_
Total Database Size: 29.16MB
Database Data Size: 20.04MB
Database Index Size: 9.12MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 1.36MB + Index: 0.75MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 1.42MB + Index: 0.42MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.06MB + Index: 0.09MB + Engine InnoDB
wp_gla_attribute_mapping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_budget_recommendations: Data: 0.19MB + Index: 0.14MB + Engine InnoDB
wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jetpack_sync_queue: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_automations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_automation_runs: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_automation_run_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_automation_run_subjects: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_automation_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_automation_versions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletters: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_scheduled_tasks: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_bounces: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.23MB + Engine InnoDB
wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscriber_tag: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_tags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_user_agents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 1.27MB + Index: 0.06MB + Engine InnoDB
wp_postmeta: Data: 0.27MB + Index: 0.25MB + Engine InnoDB
wp_posts: Data: 0.14MB + Index: 0.06MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.13MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.08MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_wc_orders_meta: Data: 0.06MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_addresses: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_operational_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_square_customers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_actionscheduler_actions: Data: 2.02MB + Index: 0.73MB + Engine InnoDB
__wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_actionscheduler_logs: Data: 1.38MB + Index: 0.45MB + Engine InnoDB
__wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_comments: Data: 0.06MB + Index: 0.09MB + Engine InnoDB
__wp_gla_attribute_mapping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_gla_budget_recommendations: Data: 0.19MB + Index: 0.14MB + Engine InnoDB
__wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_automations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_automation_runs: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
__wp_mailpoet_automation_run_logs: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_mailpoet_automation_run_subjects: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_automation_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_automation_versions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_newsletters: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_scheduled_tasks: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_statistics_bounces: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
__wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
__wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.23MB + Engine InnoDB
__wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_subscriber_tag: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_mailpoet_tags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_user_agents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_options: Data: 2.09MB + Index: 0.06MB + Engine InnoDB
__wp_postmeta: Data: 0.25MB + Index: 0.23MB + Engine InnoDB
__wp_posts: Data: 0.16MB + Index: 0.06MB + Engine InnoDB
__wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_usermeta: Data: 0.13MB + Index: 0.03MB + Engine InnoDB
__wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
__wp_wc_admin_note_actions: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
__wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_orders: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
__wp_wc_orders_meta: Data: 0.05MB + Index: 0.09MB + Engine InnoDB
__wp_wc_order_addresses: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_order_operational_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
__wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
__wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_sessions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
__wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
__wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_woocommerce_square_customers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
__wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
__wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 34
customize_changeset: 9
custom_css: 1
mailpoet_page: 1
nav_menu_item: 11
page: 9
post: 1
product: 27
product_variation: 32
revision: 32
shop_coupon: 6
shop_order: 15
shop_order_placehold: 2
wp_global_styles: 1
wp_navigation: 1
wp_template: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (11) ###

Akismet Anti-spam: Spam Protection: by Automattic - Anti-spam Team – 5.3.1
Google Listings and Ads: by WooCommerce – 2.5.16
Jetpack Protect: by Automattic - Jetpack Security team – 2.0.0
Jetpack: by Automattic – 13.1-a.7
Loco Translate: by Tim Whitlock – 2.6.6
WooCommerce Stripe Gateway: by WooCommerce – 7.9.1
WooPayments: by Automattic – 7.1.0
WooCommerce PayPal Payments: by WooCommerce – 2.5.1
WooCommerce Shipping & Tax: by WooCommerce – 2.4.2
WooCommerce Square: by WooCommerce – 4.4.1
WooCommerce: by Automattic – 8.5.2

### Inactive Plugins (6) ###

Code Snippets: by Code Snippets Pro – 3.6.2
MailPoet: by MailPoet – 4.42.0
Query Monitor: by John Blackbourn – 3.15.0
User Roles Editor for WooCommerce: by FME ADDONS – 1.0.8
WooCommerce Blocks: by Automattic – 11.7.0
WP Mail Logging: by WP Mail Logging Team – 1.12.0

### Dropin Plugins (3) ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class (Drop-in)
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: left_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to Woo.com: –
Enforce Approved Product Download Directories: ✔
HPOS feature screen enabled: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –

### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.5.4
Author URL: https://woo.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### WooPayments ###

Version: 7.1.0
Connected to WPCOM: Yes
WPCOM Blog ID: 226468543
Account ID: acct_1OI81EC8WnAGsBn1
Payment Gateway: Enabled
Test Mode: Enabled
Enabled APMs: card
WooPay: Enabled (product,cart,checkout)
WooPay Incompatible Extensions: No
Apple Pay / Google Pay: Enabled (product,cart,checkout)
Fraud Protection Level: basic
Multi-currency: Enabled
Public Key Encryption: Disabled
Auth and Capture: Enabled
Documents: Disabled
Logging: Enabled

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: CA
WooCommerce currency supported: ✔
Advanced Card Processing available in country: ✔
Pay Later messaging available in country: –
Webhook status: –
PayPal Vault enabled: –
ACDC Vault enabled: –
Logging enabled: –
Reference Transactions: ✔
Used PayPal Checkout plugin: –
Subscriptions Mode: Disabled

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-variation-management
product-virtual-downloadable
product-external-affiliate
product-grouped
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: customize-store
minified-js
new-product-management-experience
product-linked
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2024-01-30 17:05:14 +00:00
Options: ✔
Notes: 78
Onboarding: completed

### Action Scheduler ###

Canceled: 3
Oldest: 2024-01-02 13:15:26 +0000
Newest: 2024-01-29 19:38:44 +0000

Complete: 2,093
Oldest: 2023-12-30 09:41:34 +0000
Newest: 2024-01-30 13:17:28 +0000

Failed: 125
Oldest: 2023-11-30 10:08:08 +0000
Newest: 2024-01-30 13:45:43 +0000

Pending: 3
Oldest: 2024-01-30 13:47:43 +0000
Newest: 2024-01-31 13:17:28 +0000


### Status report information ###

Generated at: 2024-01-30 13:45:50 +00:00
```

</details>

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by nefeline** [comment]
Hi @carolframen: thanks for reaching out to us, and thank you for reporting this bug. I'm going ahead and moving it to our backlog so we can work on a fix.

**Comment by aprilfast** [comment]
Hi, just checking in on the status of this. I'm not really sure how to use GitHub correctly, so I hope I'm not being a pest.

**Comment by carolframen** [comment]
8554644-zen

**Comment by haszari** [comment]
In block-based themes (FSE) any links in the description are not rendered! So that's another inconsistency, maybe worth reporting a separate bug. 


#### Storefront - links, can read all the text

![Image](https://github.com/user-attachments/assets/57cf8341-8d77-4a65-b3e6-fbc842658fe9)


#### Twenty twenty four - no links, text truncated

![Image](https://github.com/user-attachments/assets/50835d1f-6a10-4bb7-81c2-8b77ead50382)

**Comment by haszari** [comment]
Looks like the text is truncated in the back end (not in css), and the block doing the rendering is `post-excerpt`, so this may need a fix in WordPress core? (investigating, dumping notes here)

![Image](https://github.com/user-attachments/assets/40d81cee-b806-4627-8651-21aaf3166c86)

**Comment by haszari** [comment]
Looks like this is actually rendered by `Product Summary` block:

https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/summary

**Comment by haszari** [comment]
This bug may be a duplicate of #44260

And there is a PR open maybe with a fix, though looks like it's stuck #48984

**Comment by haszari** [comment]
Closing as a duplicate, #44260 is the same issue and has more context & info. 

If I've missed something please reopen 😃
